FDK17プロジェクト/コード/03.サウンド/CSound.cs

```diff
@@ -1479,8 +1546,9 @@ private bool bBASSサウンドである
 		}
 		private int _n位置 = 0;
 		private int _n位置db;
-		private int _n音量 = 100;
-		private int _n音量db;
+		private int _gain = DefaultGain;
```

Comment:
Notice how we no longer even need a _n音量 field. It was used inconsistently depending on sound device type, but isn't even needed at all now because the _n音量 value never needs to be read back.